### ISSUE TITLE
install problem

### ISSUE THREAD

**Issue by joeharr4** [ExoCTK/exoctk#36]
I installed anaconda with Python 3.6, updated with

conda install numpy

set PYTHONPATH and PATH to have the anaconda install at the start of each and ran:

conda env create -f exoctk.yml

This produced the following error:

Using Anaconda API: https://api.anaconda.org
An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Current conda install:

               platform : linux-64
          conda version : 4.3.22
       conda is private : False
      conda-env version : 4.3.22
    conda-build version : not installed
         python version : 3.6.1.final.0
       requests version : 2.14.2
       root environment : /home/jh/ast/esp01/anaconda3  (writable)
    default environment : /home/jh/ast/esp01/anaconda3
       envs directories : /home/jh/ast/esp01/anaconda3/envs
                          /home/jh/.conda/envs
          package cache : /home/jh/ast/esp01/anaconda3/pkgs
                          /home/jh/.conda/pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/linux-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/linux-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/linux-64
                          https://repo.continuum.io/pkgs/pro/noarch
            config file : None
             netrc file : None
           offline mode : False
             user-agent : conda/4.3.22 requests/2.14.2 CPython/3.6.1 Linux/4.4.0-83-generic debian/stretch/sid glibc/2.23    
                UID:GID : 4419:4043

`$ /home/jh/ast/esp01/anaconda3/bin/conda-env create -f exoctk.yml`

    Traceback (most recent call last):
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/conda/exceptions.py", line 632, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/conda_env/cli/main_create.py", line 78, in execute
        directory=os.getcwd())
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/conda_env/specs/__init__.py", line 20, in detect
        if spec.can_handle():
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/conda_env/specs/yaml_file.py", line 14, in can_handle
        self._environment = env.from_file(self.filename)
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/conda_env/env.py", line 80, in from_file
        return from_yaml(yamlstr, filename=filename)
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/conda_env/env.py", line 68, in from_yaml
        data = yaml.load(yamlstr)
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/ruamel_yaml/main.py", line 75, in load
        return loader.get_single_data()
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/ruamel_yaml/constructor.py", line 60, in get_single_data
        node = self.get_single_node()
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/ruamel_yaml/composer.py", line 53, in get_single_node
        document = self.compose_document()
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/ruamel_yaml/composer.py", line 76, in compose_document
        self.get_event()
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/ruamel_yaml/parser.py", line 136, in get_event
        self.current_event = self.state()
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/ruamel_yaml/parser.py", line 215, in parse_document_end
        token = self.peek_token()
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/ruamel_yaml/scanner.py", line 144, in peek_token
        self.fetch_more_tokens()
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/ruamel_yaml/scanner.py", line 239, in fetch_more_tokens
        return self.fetch_value()
      File "/home/jh/ast/esp01/anaconda3/lib/python3.6/site-packages/ruamel_yaml/scanner.py", line 598, in fetch_value
        self.get_mark())
    ruamel_yaml.scanner.ScannerError: mapping values are not allowed here
      in "<unicode string>", line 309, column 24:
          <!-- blob contrib key: blob_contributors:v21:4038b58e ... 
                               ^

Maybe this is a conda bug?  They seem to say it is at the top.

Thanks,

--jh--

**Comment by exowanderer** [comment]
I cannot speak to what caused the issue; but I can say that I ran `conda-env create -f exoctk.yml` in my ETEO_JWST directory, successfully created the environment, and installed all of the requirements.

This could be an anaconda or otherwise local issue.  I'll go try it on a linux machine too.

For comparison, here is my `conda info`:

```
Current conda install:

               platform : osx-64
          conda version : 4.3.22
       conda is private : False
      conda-env version : 4.3.22
    conda-build version : not installed
         python version : 3.6.1.final.0
       requests version : 2.18.1
       root environment : /Users/$USER/anaconda3  (writable)
    default environment : /Users/$USER/anaconda3
       envs directories : /Users/$USER/anaconda3/envs
                          /Users/$USER/.conda/envs
          package cache : /Users/$USER/anaconda3/pkgs
                          /Users/$USER/.conda/pkgs
           channel URLs : http://ssb.stsci.edu/astroconda/osx-64
                          http://ssb.stsci.edu/astroconda/noarch
                          https://repo.continuum.io/pkgs/free/osx-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/osx-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/osx-64
                          https://repo.continuum.io/pkgs/pro/noarch
            config file : /Users/$USER/.condarc
             netrc file : None
           offline mode : False
             user-agent : conda/4.3.22 requests/2.18.1 CPython/3.6.1 Darwin/15.6.0 OSX/10.11.6    
                UID:GID : 1960:1031
```

**Comment by hover2pi** [comment]
This may have been resolved with #127 .
